evm/verify-contract.js

```diff
@@ -274,17 +271,17 @@ async function processCommand(config, chain, options) {
                 [implementation, chain.contracts.InterchainTokenService.deployer, setupParams],
                 {
                     ...verifyOptions,
-                    contractPath: 'contracts/proxies/InterchainTokenServiceProxy.sol:InterchainTokenServiceProxy',
+                    contractPath: 'contracts/proxies/Proxy.sol:Proxy',
```

Comment:
This does not work, please let me know if anyone has suggestions on how to verify the proxy here and below

Reply:
Implemented [here](https://github.com/axelarnetwork/interchain-token-service/pull/210)